official-antistasi-community/A3-Antistasi
# Western Sahara Templates

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
Currently
The Tura tribes - rebel faction
SFIA - Occupiers with outdated & imported CSAT equipment
CSAT North African Allies- Invaders, using older weapons as standard issue
NATO Desert - Desert camo & standard NATO equipment, sometimes guest starring CTRG group 4
~~UNA - Peacekeepers with no qualms about direct action, uses mostly western equipment.~~
ION PMC - Private military, uses civilian and mixed east and west military vehicles and their own small arms

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Load Western Sahara cdlc, load the templates.
Alt: While not playable, it should be **testable** with the Western Sahara compatibility mod.

********************************************************
Notes:
Known issues: 
~~I don't have too much of an idea of what kind of SF the SFIA should have.~~
~~The SFIA marker will throw an can't find texture error if loaded without WS; waiting for changes from #2885~~
